model/patch/patch.go

```diff
@@ -1228,7 +1218,7 @@ func GetCollectiveStatus(statuses []string) string {
 	} else if hasFailure {
 		return evergreen.PatchFailed
 	} else if hasAborted {
-		return evergreen.PatchAborted
+		return evergreen.VersionAborted
```

Comment:
This is kind of weird to put a version status in a patch function, and I suspect someone will one day think this is a bug (it looks like one). Can we just make a `evergreen.PatchAborted` status and set it to the same string value as `evergreen.VersionAborted`? We can just translate it to the version status if needed in `PatchStatusToVersionStatus`.

Reply:
Yeah that's fair